src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
 
 export const CuriosityMap: React.FunctionComponent = () => {
+  const [date, setDate] = useState('0')
```

Comment:
Semicolon at end of line:
```suggestion
  const [date, setDate] = useState('0');
```